docs/intro.md

```diff
@@ -49,7 +49,7 @@ And here is the **generated** Plant UML diagram for the model:
 
 🔎 Runtime introspection of the model using a powerful set of APIs
 
-🎛 Convert the model to [14+ common data model formats](./category/code-generation)
+🎛 Convert the model to [14+ common data model formats](https://concerto.accordproject.org/docs/category/code-generation)
```

Comment:
This won't work for local or PR builds. Does it work if you replace the leading `.` with `/docs`?

Reply:
Yes, i agree it won't work for local or PR builds. Here are two cases:
1) One link takes to /docs/intro/
2) Other link takes to /docs
So if we do . with /docs, then would work for second case but fail for the first case as then it would become /docs/docs

Is it possible to have same path for both when you go from learn tutorial or from Quick tutorial? 

I would say lets fix for now, so there is no broken link.. In parallel, can see how to fix this at application level by correcting paths.